jangjuseong/services/userService.js

```diff
@@ -0,0 +1,48 @@
+const userDao = require('../models/userDao');
+
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+
+const SECRET_KEY = process.env.SECRET_KEY;
+
+const signUp = async (name, email, password, profileImage) => {
+  const saltRounds = 12;
+  const hashPassword = await bcrypt.hash(password, saltRounds);
+
+  const [userInfo] = await userDao.getUser(email);
+
+  if (userInfo.email == email) {
```

Comment:
이미 `userDao.getUser()`함수가 email로 특정 유저 정보를 찾아오고 있기 떄문에 다시 email 값을 비교하지 않아도 될 것 같습니다!

```suggestion
  if ( userInfo.email ) {
```